.buildkite/lint.rayci.yml

```diff
@@ -23,6 +23,7 @@ steps:
   - label: ":lint-roller: lint: {{matrix}}"
     tags:
       - oss
+      - any_change
```

Comment:
maybe change this tag to `docscheck` or something? and just skip `docscheck` when we need to?

otherwise, this is effectively an `always`, and it feels a bit weird that it still requires `oss` as a tag.

Reply:
oh there is an always tag, i didn't know about it